test/components/results/candidate-results/personality-dimensions.test.js

```diff
@@ -66,7 +66,7 @@ describe("PersonalityDimensions", () => {
   });
 
   it("renders details if columns disabled", () => {
-    props.getOption.mockReturnValue(["PersonalityDimensionColumns"]);
+    props.getOption.mockReturnValueOnce(["PersonalityDimensionColumns"]);
```

Comment:
Why did this change?

Reply:
Because getOption was being called twice and returning this same value, once in the PersonalityDimensions component and then again in PersonalityDimensionDetails

Reply:
I think a solution here would be to mock out the sub components like https://github.com/traitify/traitify-widgets/blob/master/test/components/results/candidate-results.test.js#L4

And then the sub components could just rely on the tests in their own test files